### ISSUE TITLE
Problems with TCGABiolinks/version of R

### ISSUE THREAD

**Issue by Kimist99** [BioinformaticsFMRP/TCGAbiolinks#132]
Hello!

With the error message below, should I uninstall R3.3.3 and install a previous version? If so, which one? I'm using RStudio. Thanks in advance!

> install.packages('GenomicDataCommons')
Installing package into ‘C:/Users/New User/Documents/R/win-library/3.3’
(as ‘lib’ is unspecified)
Warning in install.packages :
  package ‘GenomicDataCommons’ is not available (for R version 3.3.3)

**Comment by tiagochst** [comment]
Hello,

I believe you are trying to install another package.

```
source("https://bioconductor.org/biocLite.R")
biocLite("GenomicDataCommons")
```

**Comment by Kimist99** [comment]
Thank you for the quick reply. I started with this:
devtools::install_github(repo = "BioinformaticsFMRP/TCGAbiolinks",ref = "GenomicDataCommons" )
And got an error that I needed to install the GenomicDataCommons package.

I just tried the code you provided above and got the same error as before about the package being incompatible with R3.3.3

 
> source("https://bioconductor.org/biocLite.R")
Bioconductor version 3.4 (BiocInstaller 1.24.0), ?biocLite for help
A new version of Bioconductor is available after installing the most recent
  version of R; see http://bioconductor.org/install
> biocLite("TCGAbiolinks")
BioC_mirror: https://bioconductor.org
Using Bioconductor 3.4 (BiocInstaller 1.24.0), R 3.3.3 (2017-03-06).
Installing package(s) ‘TCGAbiolinks’
Warning: package ‘TCGAbiolinks’ is in use and will not be installed
installation path not writeable, unable to update packages: boot, cluster,
  foreign, lattice, MASS, Matrix, mgcv, rpart, survival
> biocLite("GenomicDataCommons")
BioC_mirror: https://bioconductor.org
Using Bioconductor 3.4 (BiocInstaller 1.24.0), R 3.3.3 (2017-03-06).
Installing package(s) ‘GenomicDataCommons’
installation path not writeable, unable to update packages: boot, cluster,
  foreign, lattice, MASS, Matrix, mgcv, rpart, survival
Warning message:
package ‘GenomicDataCommons’ is not available (for R version 3.3.3)
